addons/binding/org.openhab.binding.konnected/README.md

```diff
@@ -0,0 +1,38 @@
+# Konnected Binding
+
+This binding is for interacting with the Konnected Module. A module which intefaces with existing home security sensors. 
+
+https://konnected.io/_
+
+
+## Supported Things
+
+This binding supports one type of thing:  a konnected module.
+
+## Discovery
+
+The binding will auto disocver konnected modules which are attached to the same network as the server running openhab via UPnP.  The binding will then create things for each module discovered which can be added.
```

Comment:
disocver -> discover

Can you also replace all occurrences of "konnected"  with "Konnected" since it is a brand name?